popolo/importers/popit.py

```diff
@@ -92,6 +101,29 @@ def get_popolo_model_class(self, model_name):
     def get_model_class(self, app_label, model_name):
         return apps.get_model(app_label, model_name)
 
+    def set(self, django_object, field_name, value):
+        try:
+            max_length = django_object._meta.get_field(field_name).max_length
+        except AttributeError as e:
+            if "has no attribute '{0}'".format(field_name) in text_type(e):
+                max_length = None
+            else:
+                raise
+        if max_length is None:
+            # Then do no more checking, just set the value:
+            setattr(django_object, field_name, value)
+        else:
+            if self.truncate == 'exception':
+                # Just set the value, if it's too long an exception will be raised:
+                setattr(django_object, field_name, value)
```

Comment:
Right, yes - trying it that way, I think it is clearer with a single `setattr` and a couple of comments.